README.md

```diff
@@ -65,18 +61,18 @@ Building with just these adapters will result in a smaller bundle which should a
 
 **Build standalone prebid.js**
 
-- Clone the repo, run `npm install`
+- Clone the repo, run `yarn install`
```

Comment:
Might want to add a link to the yarn project for those that are unfamiliar with it.